src/index.js

```diff
@@ -17,6 +18,6 @@ const comment = document.createComment("\n"+hiringMessage.toString()+"\n");
 document.insertBefore(comment, document.firstChild);
 
 ReactDOM.render(
-    <PreRegister/>,
+    <Site/>,
```

Comment:
Why not put contents of `Site` here in index.js? Seems like an extra empty shell component

Reply:
Wasn't sure if we were going to archive the pre-register site in a folder. If so, the shell component was to make it easier to swap.